tools/nni_cmd/constants.py

```diff
@@ -81,4 +81,15 @@
 
 COLOR_GREEN_FORMAT = '\033[1;32;32m%s\033[0m'
 
-COLOR_YELLOW_FORMAT = '\033[1;33;33m%s\033[0m'
\ No newline at end of file
+COLOR_YELLOW_FORMAT = '\033[1;33;33m%s\033[0m'
+
+ModuleName = {
+    'TPE': 'nni.hyperopt_tuner.hyperopt_tuner',
+    'Random': 'nni.hyperopt_tuner.hyperopt_tuner',
+    'Anneal': 'nni.hyperopt_tuner.hyperopt_tuner',
+    'Evolution': 'nni.evolution_tuner.evolution_tuner',
+    'SMAC': 'nni.smac_tuner.smac_tuner',
+    'BatchTuner': 'nni.batch_tuner.batch_tuner',
```

Comment:
Seems this part has conflict with master now

Reply:
You mean the customer tuner?